app/scripts/lib/crypto/base32.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const base32Decode = require('base32-decode');
+
+function importFxaCryptoDeriver() {
+  return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
+}
+
+// Base32 encoding based on Douglas Crockford
+// Ref: https://www.crockford.com/wrmg/base32.html
+const ALPHABET_BASE32 = '0123456789ABCDEFGHJKMNPQRSTVWXYZ';
```

Comment:
did we want to use lowercase here? just like the db mysql change?

Reply:
I'll hold off on that change since it will make the test hashes a bit different. Once this merges I can make that change in https://github.com/mozilla/fxa-content-server/pull/6203.